subworkflows/local/dram_distill_swf.nf

```diff
@@ -0,0 +1,41 @@
+include { DRAM_DISTILL as DRAM_DISTILL_PER_ASSEMBLY    } from '../../modules/ebi-metagenomics/dram/distill/main'
+include { DRAM_DISTILL as DRAM_DISTILL_PER_SAMPLESHEET } from '../../modules/ebi-metagenomics/dram/distill/main'
+include { SUMMARISE_FOR_DRAM_INPUT                     } from '../../modules/local/summarise_for_dram_input'
+
+workflow DRAM_DISTILL_SWF {
+
+    take:
+    ch_contigs           // tuple [meta, path(contigs_fasta)]
+    ko_per_contigs_tsv   // tuple [meta, path(tsv)]
+    interproscan_tsv     // tuple [meta, path(tsv)]
+    dbcan_overview       // tuple [meta, path(tsv)]
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    SUMMARISE_FOR_DRAM_INPUT(
+        ch_contigs.join( ko_per_contigs_tsv ).join( interproscan_tsv ).join( dbcan_overview ),
+    )
```

Comment:
The module expects the proteins faa of the assembly in fasta format as input

Reply:
yup. fixed